drivers/kscan/kscan_mchp_xec.c

```diff
@@ -0,0 +1,408 @@
+/*
+ * Copyright (c) 2019 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+#include <device.h>
+#include <drivers/kscan.h>
+#include <kernel.h>
+#include <soc.h>
+#include <sys/atomic.h>
+#include <logging/log.h>
+
+#define LOG_LEVEL CONFIG_KSCAN_LOG_LEVEL
+LOG_MODULE_REGISTER(kscan_mchp_xec);
```

Comment:
same here `LOG_MODULE_REGISTER(kscan_mchp_xec, CONFIG_KSCAN_LOG_LEVEL);`
as you used it here it won't be applied because LOG_LEVEL must be defined before including `log.h`. This is legacy mode of registering module with log level. Recommended one is `LOG_MODULE_REGISTER(name,level)`

Reply:
@nordic-krch I'll do the change later this week. Thanks for the explanation